packages/parser/src/grammar.spec.ts

```
@@ -0,0 +1,208 @@
+/**
+ * Tests for Grammar Definition Block parsing (Phase 1.6)
+ */
+
+import { describe, expect, test } from "bun:test";
+import { parse } from "tpeg-core";
+
+// Helper function for easier testing
+const testParse = <T>(parser: any, input: string) => parse(parser)(input);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)